src/utils/strings/englishConstants.tsx

```diff
@@ -552,6 +552,32 @@ const englishConstants = {
     SORT_BY_LATEST_PHOTO: 'recent photo',
     SORT_BY_MODIFICATION_TIME: 'last updated',
     SORT_BY_COLLECTION_NAME: 'album name',
+    FIX_LARGE_THUMBNAILS: 'fix large thumbnails',
+    THUMBNAIL_REPLACED: 'thumbnail replaced',
+    FIX: 'fix',
+    FIX_LATER: 'fix later',
+    REPLACE_THUMBNAIL_NOT_STARTED: () => (
+        <>
+            Some of your files have been upload with very large thumbnail due to
```

Comment:
`some of your videos thumbnails can be compressed to save space. would you like ente to compress them?`